app/assets/stylesheets/modules/_nav.styl

```diff
@@ -1,6 +1,6 @@
 nav.top-nav
   top 0
-  z-index 100
+  z-index 10000
```

Comment:
What's this for? If this is necessary to make the styling work as intended, let's fix the underlying cause rather than punch up the z-index higher.